<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChartBuilder</class>
 <widget class="QMainWindow" name="ChartBuilder">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChartBuilder</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myfiles.qrc">
    <normaloff>:/new/prefix1/ChartBuilder.png</normaloff>:/new/prefix1/ChartBuilder.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>735</width>
      <height>440</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>20</y>
      <width>143</width>
      <height>348</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="formula">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Input formula&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="formulaInput"/>
     </item>
     <item>
      <widget class="QLabel" name="points">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Input points&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="X">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;X =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="X1">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="X2">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="Y">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Y =&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Y1">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="Y2">
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="DrawGraph">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;Build graph&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 85, 255);
font: 75 8pt &quot;System&quot;;</string>
       </property>
       <property name="text">
        <string>Draw</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="LineThickness">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change line thickness&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 pt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 pt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 pt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 pt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 pt</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6 pt</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="LineType">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change color&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SolidLine</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/solid.png</normaloff>:/new/prefix1/solid.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashLine</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/dash.png</normaloff>:/new/prefix1/dash.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DotLine</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/dot.png</normaloff>:/new/prefix1/dot.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotLine</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/dashdot.png</normaloff>:/new/prefix1/dashdot.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DashDotDotLine</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/dashdotdot.png</normaloff>:/new/prefix1/dashdotdot.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="Color">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Change line type&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Black</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/black.png</normaloff>:/new/prefix1/black.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Red</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/red.png</normaloff>:/new/prefix1/red.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Green</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/green.png</normaloff>:/new/prefix1/green.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Blue</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/blue.png</normaloff>:/new/prefix1/blue.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cyan</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/cyan.png</normaloff>:/new/prefix1/cyan.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Magenta</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/magenta.png</normaloff>:/new/prefix1/magenta.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Yellow</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/yellow.png</normaloff>:/new/prefix1/yellow.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gray</string>
        </property>
        <property name="icon">
         <iconset resource="myfiles.qrc">
          <normaloff>:/new/prefix1/gray.png</normaloff>:/new/prefix1/gray.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Change">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Edit built graph&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 0, 255);
font: 75 8pt &quot;System&quot;;</string>
       </property>
       <property name="text">
        <string>Change</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Delete">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Delete graph&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 0);
font: 75 8pt &quot;System&quot;;</string>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Save">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Save as a picture&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 170, 0);
font: 75 8pt &quot;System&quot;;</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Exit">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exit application&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);
font: 75 8pt &quot;System&quot;;</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>935</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="myfiles.qrc">
     <normaloff>:/new/prefix1/save.png</normaloff>:/new/prefix1/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save as a picture</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="myfiles.qrc">
     <normaloff>:/new/prefix1/delete.png</normaloff>:/new/prefix1/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete graphs</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="myfiles.qrc">
     <normaloff>:/new/prefix1/exit.png</normaloff>:/new/prefix1/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myfiles.qrc"/>
 </resources>
 <connections/>
</ui>
